modules/consul_cluster/scripts/install_hashitools_consul_client.sh.tpl

```diff
@@ -15,20 +15,39 @@ INSTANCE_ID=`curl -s http://169.254.169.254/latest/meta-data/instance-id`
 AVAILABILITY_ZONE=`curl -s http://169.254.169.254/latest/meta-data/placement/availability-zone`
```

Comment:
Two thoughts on this: 

1.) Consider using `$(curl)` instead of back ticks
2.) Consider using longform arguments to make it clear that `-s` is `--silent`

Reply:
```suggestion
AVAILABILITY_ZONE=$(curl -silent http://169.254.169.254/latest/meta-data/placement/availability-zone)
```

Reply:
Does that look good @ksatirli ?

Reply:
Absolutely!